src/Symfony/Bundle/SecurityBundle/Resources/config/security.xml

```diff
@@ -26,6 +26,19 @@
         </service>
         <service id="Symfony\Component\Security\Core\Authentication\Token\Storage\TokenStorageInterface" alias="security.token_storage" />
 
+        <service id="security.helper" class="Symfony\Component\Security\Core\Security">
+            <argument type="service">
+                <service class="Symfony\Component\DependencyInjection\ServiceLocator">
+                    <tag name="container.service_locator" />
+                    <argument type="collection">
```

Comment:
You should make the Security class implement ServiceSubscriberInterface instead. But maybe you prefer no dep on the DI component?

Reply:
Yea... that was the basic idea: it felt better to keep the `Security` class from the component coupled only to `psr/container`. And actually, that wasn't listed as a dep in `composer.json`. I'm going to add that now... tests must've passed because another dep of `symfony/security` already required it?

Reply:
Haha, or because I don't have a test for that class... fixing that now :)